libraries/AC_PrecLand/AC_PrecLand_Backend.h

```diff
@@ -1,8 +1,12 @@
 #pragma once
 
+#include "AC_PrecLand.h"
```

Comment:
```suggestion
#include "AC_PrecLand_config.h"
```

... and leave the AC_PrecLand.h include in-place within the `#if`